libpod/container.go

```diff
@@ -606,6 +609,15 @@ func (c *Container) Init() (err error) {
 	g.AddBindMount(runDirResolv, "/etc/resolv.conf", []string{"rw"})
 	// Bind mount hosts
 	g.AddBindMount(runDirHosts, "/etc/hosts", []string{"rw"})
+
+	uid, gid, err := c.ParseUser(c.config.User)
```

Comment:
We should only do this if c.config.User is not "" - if it's the empty string they never specified a user, so we can use use whatever is in the spec

Reply:
chrootuser.GetUser returns 0,0, nil if the user is "".